### PR TITLE
[API] add endpoint to re-verify an entire dataset

### PR DESCRIPTION
**Pull request by jbygdell**
**Description**
This PR build upon #1120 and add the option to re-verify all files in a dataset

### CODE REVIEW

File: sda/cmd/api/api.md [hunk 1]
```diff
@@ -105,6 +105,22 @@ Admin endpoints are only available to a set of whitelisted users specified in th
     curl -H "Authorization: Bearer $token" -X POST  https://HOSTNAME/dataset/release/my-dataset-01
     ```
 
+- `/dataset/verify/*dataset`
+  - accepts `PUT` requests with the dataset name as last part of the path`
```
Comment (MalinAhlberg) [inline sda/cmd/api/api.md:109]:
```suggestion
  - accepts `PUT` requests with the dataset name as last part of the path
```

**Review by MalinAhlberg — APPROVED** [review]
:+1:

**Review by nanjiangshu — COMMENTED** [review]
It looks good except for some minor comments that needs to be fixed.

**Review by nanjiangshu — APPROVED** [review]
Looks good!

**Pull request merged**